setup.sh

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env bash
+
+# Bash strict mode
+set -euo pipefail
+
+echo "PORT=8083" > .env
+echo "EXTRA_PORT='8082'" >> .env
+echo "WEBPACK_DEV_URL=http://0.0.0.0:8082" >> .env
+
+echo "##########################"
+echo "#                        #"
+echo "#         Github         #"
```

Comment:
The proper capitalisation is "GitHub", and this should be corrected throughout this PR.